src/microbe_stage/MicrobeInternalCalculations.cs

```diff
@@ -796,4 +787,31 @@ private static Vector3 ChooseFromSymmetricFlagella(IReadOnlyList<OrganelleTempla
 
         return maximumMovementDirection;
     }
+
+    private static void AddSpecificCapacity(IReadOnlyList<OrganelleTemplate> organelles,
+        Dictionary<Compound, float> capacities, float totalNominalCap)
+    {
+        var count = organelles.Count;
+
+        // Update the variant of this logic in UpdateSpecificCapacities if changes are made
+        for (int i = 0; i < count; ++i)
+        {
+            var organelle = organelles[i];
+
+            var specificCapacity = GetAdditionalCapacityForOrganelle(organelle.Definition, organelle.Upgrades);
+
+            if (specificCapacity.Compound == Compound.Invalid)
+                continue;
+
+            // If this is updated the code in CompoundBag.AddSpecificCapacityForCompound must also be updated
+            if (capacities.TryGetValue(specificCapacity.Compound, out var currentCapacity))
+            {
+                capacities[specificCapacity.Compound] = currentCapacity + specificCapacity.Capacity;
+            }
+            else
+            {
+                capacities.Add(specificCapacity.Compound, specificCapacity.Capacity + totalNominalCap);
+            }
```

Comment:
```suggestion
            capacities.TryGetValue(specificCapacity.Compound, out var currentCapacity);
            capacities[specificCapacity.Compound] = currentCapacity + specificCapacity.Capacity;
```
Can't this be simplified as that?